sdv/constraints/base.py

```diff
@@ -197,6 +213,22 @@ def filter_valid(self, table_data):
 
         return table_data[valid]
 
+    def _validate_constraint_columns(self, table_data):
+        """Return True if all required columns are in ``table_data``.
+
+        Args:
+            table_data (pandas.DataFrame):
+                Table data.
+
+        Returns:
+            bool:
+                Whether or not all column names in ``constraint_columns`` are present
+                in ``table_data``. Returns False if ``constraint_columns`` is None or empty.
+        """
+        if not self._constraint_columns:
```

Comment:
If we change the default value to a tuple we can skip this check and just do the `all` check below.